backends/ubpf/tests/testdata/oko-test-actions.p4

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
what is the difference between the program in examples and this one?

Reply:
if they are the same then it would be nice to avoid the duplication. symlinks?

Reply:
We've added symlinks.